spark-cassandra-connector-java/src/it/scala/com/datastax/spark/connector/CassandraJavaUtilSpec.scala

```diff
@@ -13,6 +13,8 @@ import scala.collection.JavaConversions._
 class CassandraJavaUtilSpec extends FlatSpec with Matchers with BeforeAndAfter with SharedEmbeddedCassandra with SparkTemplate {
 
   useCassandraConfig("cassandra-default.yaml.template")
+  useSparkConf()
```

Comment:
I think it would be much more readable to explicitly state which SparkConf to use. If the only purpose of this method is to make sure the default SparkConf is used, `useSparkConf(SparkTemplate.default)` would communicate this intent much more clearly.